pkgs/top-level/all-packages.nix

```diff
@@ -6367,6 +6367,11 @@ in
 
   trilium = callPackage ../applications/office/trilium { };
 
+  # trilinos = callPackage ../development/libraries/science/math/trilinos {
```

Comment:
Should this be here / be uncommented?

Reply:
Yeah this should be removed. @wd15 said that trillinos will not work with fipy so not need to add it.

Reply:
Removed this from commit history now